### ISSUE TITLE
aws-vault v5.0.0 removes default profile block header

### ISSUE THREAD

**Issue by mbacchi** [99designs/aws-vault#475]
When adding a new credential to `aws-vault` on v5.0.0, it removes the `[default]` block header.

```
$ aws-vault --version
v5.0.0
$ aws-vault list
Profile                  Credentials              Sessions                 
=======                  ===========              ========                 
default                  default (missing)        -                        
user1                    user1                    1577058481 (mfa) 
$ cat ~/.aws/config 
[default]
output=json
region=us-east-1

[profile user1]
mfa_serial=arn:aws:iam::XXX:mfa/user1
region=us-east-2

$ aws-vault add experiment_user1
Enter Access Key ID: XXX
Enter Secret Access Key: XXX
Added credentials to profile "experiment_user1" in vault

$ cat ~/.aws/config 
output=json
region=us-east-1

[profile user1]
mfa_serial=arn:aws:iam::XXX:mfa/user1
region=us-east-2

[profile experiment_user1]


```

**Comment by ZoidBB** [comment]
I had this issue a couple days ago, can confirm. I'm going to look into what's causing this.

**Comment by ZoidBB** [comment]
The INI lib being used doesn't put the header over default values by default, so this ended up being a one-line change to configure it to output the header.

I've created PR #476 to address this.

**Comment by mtibben** [comment]
Fixed in #476
